### ISSUE TITLE
downloadFrameworks.sh doesnot download all xcframework

### ISSUE THREAD

**Issue by EnjoyWT** [holzschu/a-shell#535]
I followed the readme step and found that there is always a lack of pythona-crypto.xxx.xcframework and python3_ Ios-Crypto.xxx.xcframework and pick.framework, libpng.xcframework. Is there any special configuration required?
I install the following steps to compile the project

1. download the entire project and its sub-modules: git submodule update --init --recursive

2. download all the xcFrameworks: downloadFrameworks.sh

3. Solve the signature problem

os： macOS Monterey， M2

<img width="709" alt="企业微信截图_c4171081-0402-4ae0-95dc-f631b7d428eb" src="https://user-images.githubusercontent.com/17864126/211246738-97e71dad-8508-4c2f-90fb-c3e867ab8e87.png">

**Comment by holzschu** [comment]
Hello, 
Thank you for pointing this. The instructions are outdated: you cannot skip the "If you're very motivated" step, you have to compile the Python frameworks, or remove them from the "Embed" step. That will take care of everything that contains "cpython/XcFrameworks" or "cpython/Python-aux". I'm going to fix that.

There will still be issues with dash and vimproc, which have been added recently and are not in `downloadFrameworks.sh` yet.

**Comment by EnjoyWT** [comment]
thank you for your wonderful tools and your efforts. Expect new instructions。

**Comment by holzschu** [comment]
Thanks again for the issue. I've updated the instructions, committed all recent changes and set up the missing frameworks.
I hope I haven't forgotten anything, please let me know if it still doesn't work. 

The short version of the new instructions is that you can chose between:
- go into the cpython directory, run `downloadAndCompile.sh`, wait a few hours, build the a-Shell project, or
- remove all python-related frameworks from the a-Shell project, build directly.

**Comment by EnjoyWT** [comment]
Hi, I have tried this "go into the cpython directory, run downloadAndCompile.sh, wait a few hours, build the a-Shell project". but it does work. it look like some files are missing. 
<img width="948" alt="企业微信截图_a142cfa9-b5a8-43d9-987a-e26d4c22ea8c" src="https://user-images.githubusercontent.com/17864126/211449555-ff5bac79-f06e-4e63-bcea-55195ff999e4.png">
Then, I removed all python-related fameworks, the a-Shell project build sucess.  
Anyway, Thanks again.
